modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -91,40 +144,51 @@ endif::[]
 
 
 ifeval::[{suma-content} == true]
-
+.Procedure: Alternative Method for Adding {ubuntu} Channels and Repositories (Unsupported)
+.  
 [WARNING]
 ====
 The ``spacewalk-utils`` package is not supported by {suse}, including the ``spacewalk-common-channels`` tool.
 You will not be supported if your system becomes inoperable by using these tools.
 ====
 
-
-
-.Procedure: Alternative Method for Adding {ubuntu} Channels (Unsupported)
-
 . At the command prompt on the {productname} Server, as root, install the [systemitem]``spacewalk-utils`` package:
 +
 ----
 zypper in spacewalk-utils
 ----
 . Add the {ubuntu} channels.
-Adjust the version of the channel names to match your {ubuntu} version:
-+
+
+For {ubuntu} 16.04:
+
+----
+spacewalk-common-channels \
+ubuntu-1604-amd64-main \
+ubuntu-1604-amd64-main-updates \
+ubuntu-1604-amd64-main-security
+----
+
+For {ubuntu} 18.04:
+
 ----
-spacewalk-common-channels ubuntu-1804-amd64-main \
+spacewalk-common-channels \
+ubuntu-1804-amd64-main \
 ubuntu-1804-amd64-main-updates \
 ubuntu-1804-amd64-main-security
 ----
-. Synchronize the new custom channels.
 
-endif::[]
+When you have the channels set up, associate each channel with the appropriate repository, and synchronize then channels (optionally configure scheduled syncronization).
 
+To do this, proceed to menu:Software[Manage > Channels], and for each channel you created, click on it and:
 
+. Navigate to the [guimenu]``Repositories``
+. Navigate to the [guimenu]``Sync``and click btn:[Sync Now] to begin synchronization immediately.
```

Comment:
```suggestion
. Navigate to the [guimenu]``Sync`` sub-tab.
. Click btn:[Sync Now] to begin synchronization immediately.
```

Reply:
I think we usually write "subtab" (w/o a dash).